.evergreen/run-mmapv1-storage-test.sh

```diff
@@ -22,6 +22,8 @@ export JAVA_HOME="/opt/java/jdk11"
 
 echo "Running tests with ${JDK}"
 ./gradlew -version
-./gradlew -PjdkHome=/opt/java/${JDK} --stacktrace --info \
--Dorg.mongodb.test.uri=${MONGODB_URI} \
--Dtest.single=RetryableWritesProseTest driver-sync:test driver-core:test
+for PACKAGE in driver-sync driver-core ; do
```

Comment:
It doesn't work to just do:

    driver-sync:test driver-core:test --tests RetryableWritesProseTest

?

Reply:
No. It will run all of `driver-sync::test` and then the two tests in `driver-core::test`.